scripts/actions/__tests__/send-and-update-translation-queue.test.js

```diff
@@ -1,23 +1,314 @@
+'use strict';
+const { describe, expect, test, beforeEach } = require('@jest/globals');
+const { when } = require('jest-when');
+
 const fs = require('fs');
+const path = require('path');
 
-const { getContent } = require('../send-and-update-translation-queue');
+process.env.DB_CONNECTION_INFO = '{}';
+const {
+  getTranslations,
+  updateTranslation,
+  deleteTranslation,
+  addJob,
+  updateJob,
+  addTranslationsJobsRecord,
+} = require('../translation_workflow/database');
+const { vendorRequest, uploadFile } = require('../utils/vendor-request');
+const {
+  getReadyToGoTranslationsForEachLocale,
+  createJobs,
+  createBatches,
+  uploadFiles,
+} = require('../send-and-update-translation-queue');
 
-jest.mock('../serialize-mdx');
+console.log = () => {};
```

Comment:
Instead of reassigning the function, you can just have Jest mock it out for you.
```suggestion
jest.spyOn(console, "log")
```

Reply:
@elaguerta mentioned using `--silent` as an option when running the tests. i like that, so i will remove this line entirely, and to suppress console output just run the tests as: `yarn test --silent`